python/Dockerfile.test

```diff
@@ -1,4 +1,7 @@
 ARG base_image
 FROM $base_image
+# There must be a better way to make the test/validate.sh script executable...
+USER root
 COPY test test
+RUN chmod ug+x test/validate.sh
```

Comment:
Mark it as executable in Git then `commit`:
```
git update-index --chmod=+x validate.sh
```

Reply:
Tried that and it didn’t work.

- executed the command — shows that it was done in the branch
- removed the `USER root` and `chmod` and got an error — no execute permission
- added back in the `chmod` and got an error — not allowed to change permissions

Seems there should be some way.  Ideas?

Reply:
If you are using Windows the issue might be at the step copying the file from the filesystem (where the executable flag is not saved) to the container. It will work if your filesystem keeps the executable flag, but it won't if it doesn't... I am not sure there is a way around this other than really - c.c. @WadeBarnes 

For reference, in `aries-vcr` we have to do this to ensure scripts work in the final images: https://github.com/bcgov/aries-vcr/blob/master/docker/utils/Dockerfile.fixpermissions